screencaptor-sample/src/test/java/com/wealthfront/screencaptor/sample/ScreenshotComposeRoboletricTest.kt

```diff
@@ -0,0 +1,104 @@
+package com.wealthfront.screencaptor.sample
+
+import android.Manifest.permission.READ_EXTERNAL_STORAGE
+import android.Manifest.permission.WRITE_EXTERNAL_STORAGE
+import android.graphics.BitmapFactory.decodeByteArray
+import android.graphics.BitmapFactory.decodeFile
+import android.os.Environment
+import androidx.annotation.RawRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.unit.dp
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
+import androidx.test.rule.GrantPermissionRule
+import com.wealthfront.screencaptor.ScreenCaptor
+import org.junit.After
+import org.junit.Assert.assertTrue
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.GraphicsMode
+import org.robolectric.shadows.ShadowPixelCopy
```

Comment:
I notice this Shadow isn't being used. Do we need it for ComposeImageCaptureHelper to work?

Reply:
Nice catch. Removed unused imports. 

ShadowPixelCopy was needed because robelectric adds it by default now